src/base-container/index.scss

```diff
@@ -13,15 +13,15 @@
 
 
 .modal-body-container .pgn__modal-body-content {
-  height: 100%;
+  height: 100% !important;
 }
```

Comment:
shouldn't this be applied only if some other "custom" class is applied? Otherwise this will override all modals in prospectus

Reply:
`.modal-body-container` is our custom class so it wont be applied to other modal bodies. 

Also, these styles were being overridden by paragon styles so I had to make them all important.